tfx/components/trainer/executor.py

```diff
@@ -290,10 +303,14 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
           export_dir_base=fn_args.eval_model_dir,
           eval_input_receiver_fn=training_spec['eval_input_receiver_fn'])
 
+      # Copy model run information to ModelRun artifact
+      io_utils.copy_dir(fn_args.serving_model_dir, fn_args.model_run_dir)
+
       absl.logging.info('Exported eval_savedmodel to %s.',
                         fn_args.eval_model_dir)
     else:
       absl.logging.info(
           'eval_savedmodel export for TFMA is skipped because '
           'this is not the chief worker.'
       )
+      
```

Comment:
revert this diff?

Reply:
i added this for the linter (which you mentioned in a previous comment)

Reply:
I mean the file ends with a \n

basically here shouldn't be any diff compare to original executor.py